ksqldb-engine/src/main/java/io/confluent/ksql/connect/supported/Connectors.java

```diff
@@ -58,15 +57,9 @@
 
   public static Map<String, String> resolve(final Map<String, String> configs) {
     final SupportedConnector connector = CONNECTORS.get(configs.get(CONNECTOR_CLASS));
-    final Map<String, String> resolvedConfigs = new HashMap<>(
+    return new HashMap<>(
         connector == null ? configs : connector.resolveConfigs(configs)
     );
-
-    // at the moment, KSQL only supports String keys - when we
-    // support non-string keys we should remove this line
```

Comment:
@agavra I've removed this line on the basis of this comment, as a minor breaking change. Let me know if this isn't what you had in mind for deprecation of this convenience feature. Thanks!